### PR TITLE
[CSL-780] Start moving GodTossing

### PR DESCRIPTION
**Pull request by neongreen**

### CODE REVIEW

File: godtossing/Pos/Ssc/GodTossing/SecretStorage.hs [hunk 1]
```diff
@@ -6,12 +6,12 @@ module Pos.Ssc.GodTossing.SecretStorage
 
 import           Universum
 
-import           Pos.Binary.Ssc           ()
-import           Pos.DB                   (MonadDB)
-import           Pos.DB.Misc.Common       (miscGetBi, miscPutBi)
-import           Pos.Ssc.GodTossing.Core  (Opening, SignedCommitment)
-import           Pos.Ssc.GodTossing.Types (GtSecretStorage (..))
-import           Pos.Types                (EpochIndex)
+import           Pos.Binary.Ssc.GodTossing.Types ()
```
Comment (gromakovsky) [inline godtossing/Pos/Ssc/GodTossing/SecretStorage.hs]: Perhaps we can get rid of `Ssc` in `Pos.Binary.Ssc.GodTossing`, especially now when `GodTossing` is in a separate package.

Comment (neongreen) [inline godtossing/Pos/Ssc/GodTossing/SecretStorage.hs]: Okay.

Comment (neongreen) [inline godtossing/Pos/Ssc/GodTossing/SecretStorage.hs]: (I think that it'd be better to get rid of it once all or most of GodTossing is moved, though.)

**Comment by neongreen** [comment]
The build failed due to OOM so I'm going to merge it with the big red button.

File: godtossing/Pos/Ssc/GodTossing/SecretStorage.hs (hunk 1 above)
Comment (neongreen) [inline godtossing/Pos/Ssc/GodTossing/SecretStorage.hs]: Ah, sorry, I didn't notice that you said `Pos.Binary.Ssc.GodTossing`. Okay, I'll get rid of it in the next PR.

**Pull request merged**